src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/NonScheduledReleaseSummary.test.tsx

```diff
@@ -108,8 +109,10 @@ describe('NonScheduledReleaseSummary', () => {
         <NonScheduledReleaseSummary
           includeCreateAmendmentControls
           release={produce(testRelease, draft => {
-            draft.permissions.canUpdateRelease = true;
-            draft.permissions.canMakeAmendmentOfRelease = true;
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            draft.permissions!.canUpdateRelease = true;
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            draft.permissions!.canMakeAmendmentOfRelease = true;
```

Comment:
All of these non-null assertions are a little nasty. Would consider creating a new extended `ReleaseWithPermissions` type where `permissions` are non-null and use it for the `testRelease`

Reply:
Also switched all the other test files to not use a variable and to use this new `ReleaseWithPermissions` type